internal/storage/image.go

```diff
@@ -465,7 +492,7 @@ func (svc *imageService) PrepareImage(inputSystemContext *types.SystemContext, i
 	return srcRef.NewImage(svc.ctx, systemContext)
 }
 
-//nolint
+// nolint
```

Comment:
Oh, we got a complaint here...

Reply:
That's my vim plugin, let me do it in a way which should be fine.